.ci/pytorch/win-test-helpers/test_libtorch.bat

```diff
@@ -34,17 +34,9 @@ set CPP_TESTS_DIR=%TMP_DIR_WIN%\build\torch\test
 :: Skip verify_api_visibility as it a compile level test
 if "%~1" == "verify_api_visibility" goto :eof
 
-:: See https://github.com/pytorch/pytorch/issues/25161
-if "%~1" == "c10_metaprogramming_test" goto :eof
 if "%~1" == "module_test" goto :eof
 :: See https://github.com/pytorch/pytorch/issues/25312
 if "%~1" == "converter_nomigraph_test" goto :eof
-:: See https://github.com/pytorch/pytorch/issues/35636
-if "%~1" == "generate_proposals_op_gpu_test" goto :eof
-:: See https://github.com/pytorch/pytorch/issues/35648
```

Comment:
Please comment on https://github.com/pytorch/pytorch/issues/35648 if you think it's no longer an issue.